app.js

```diff
@@ -31,7 +31,7 @@ app.set('trust proxy', 1)
 app.use(
   rateLimiter({
     windowMs: 15 * 60 * 1000,
-    max: 100,
+    max: 1000,
     message: 'Too many requests, please try again later'
   })
 )
```

Comment:
I'm not sure if it's just something on my end but I keep getting a "Too many requests" error whenever I try testing your app. I always have to comment out the middleware. Please let me know if there are any extra settings missing on my end or review these properties to avoid that frequent error.